packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -302,18 +331,35 @@ export class AtomicCategoryFacet
 
     const nonActiveParents = this.facetState.parents.slice(0, -1);
     const activeParent = this.facetState.parents.slice(-1)[0];
+    const activeParentDisplayValue = getFieldValueCaption(
+      this.field,
+      activeParent.value,
+      this.bindings.i18n
+    );
 
     return (
       <ul part="parents" class="mt-3">
         {this.renderAllCategories()}
         {nonActiveParents.map((parent) => this.renderParent(parent))}
-        <li part="active-parent" class="parent-active">
-          {getFieldValueCaption(
-            this.field,
-            activeParent.value,
-            this.bindings.i18n
-          )}
-        </li>
+        <FacetValueLink
+          displayValue={activeParentDisplayValue}
+          numberOfResults={activeParent.numberOfResults}
+          isSelected={true}
+          i18n={this.bindings.i18n}
+          onClick={() => {
+            this.activeValueFocus.focusAfterSearch();
+            this.facet.deselectAll();
+          }}
+          searchQuery={this.facetState.facetSearch.query}
+          part="active-parent"
```

Comment:
This change is technically slightly breaking, since the part is no longer on a `li` element. I don't think it's significant though.

Reply:
Actually haha this was important my bad!